addons/hr_expense/static/src/scss/hr_expense.scss

```diff
@@ -14,7 +14,7 @@
             height: 230px;
             background: transparent url(/hr_expense/static/img/nocontent.png) no-repeat center;
             background-size: 300px 230px;
-            margin-bottom: 0.75rem;
+            margin-bottom: $spacer * .75;
```

Comment:
commit e6ce755a1b06799a9f670cba07f6a5592e7ba1ab
The parent element already have the margin from the <p> tag at 1rem, I think this margin-bottom can be removed since it doesn't apply any visual difference without it.
![image](https://github.com/odoo/odoo/assets/118886338/40439edf-3b8b-458b-9e21-0d89a9a32a78)
![image](https://github.com/odoo/odoo/assets/118886338/4ae729fb-d65d-43bf-b357-d71fe734aabf)